lib/twingly/url.rb

```diff
@@ -123,7 +127,7 @@ def normalized_path
     end
 
     def valid?
-      addressable_uri && public_suffix_domain && SCHEMES.include?(normalized_scheme)
+      !!(scheme =~ ACCEPTED_SCHEMES)
```

Comment:
It looks funny, seems like we only require a scheme to be valid, but I now that's not the case (this is the same as before). But I just wanted to write something about it.

I think making what we consider being an valid URL even more clear would be great. That might be future tasks for @twingly-mob.